### ISSUE TITLE
The eUICC Firmware version information extracted from EUICCInfo2 is being incorrectly displayed on the EasyLPAC tool.

### ISSUE THREAD

**Issue by BhushanPatil555** [creamlike1024/EasyLPAC#25]
When the euiccFirmwareVer[3].VersionType value is "83 03 42 90 00", the tool incorrectly interprets the "42 90 00" part as a hexadecimal number and converts it to a decimal value. This leads to an inaccurate display of the firmware version.

Expected behavior is 42.90.00

**Comment by creamlike1024** [comment]
Have you enabled debug output? EasyLPAC is just an LPAC GUI frontend, any issues related to eUICC interaction should be reported to [lpac](https://github.com/estkme-group/lpac)

**Comment by BhushanPatil555** [comment]
Yes. I will report it to [lpac](https://github.com/estkme-group/lpac).
Thanks
